src/model/attributes/relations/MorphOne.ts

```diff
@@ -0,0 +1,100 @@
+import { Schema as NormalizrSchema } from 'normalizr'
+import { Schema } from '../../../schema/Schema'
+import { Element, Collection } from '../../../data/Data'
+import { Query } from '../../../query/Query'
+import { Model } from '../../Model'
+import { Relation, Dictionary } from './Relation'
+
+export class MorphOne extends Relation {
+  /**
+   * The field name that contains id of the parent model.
+   */
+  protected id: string
+
+  /**
+   * The field name that contains type of the parent model.
+   */
+  protected type: string
```

Comment:
Following the API already established in Vuex ORM found in the following docs: https://github.com/vuex-orm/vuex-orm/blob/master/src/attributes/relations/MorphOne.ts

I personally would be open to renaming these two fields to make them a little more explicit so that the comments are a little less necessary.

Would you favour maintaining the previous interface or renaming these to something a bit more explicit?

Reply:
Good point! This is internal stuff so I think it would be totally OK to rename it.

In Laravel, it's named `morphType` and `morphClass`. So maybe `morphId` and `morphType` might be easier to read?

Reply:
Thanks for the feedback! I think `morphId` and `morphType` will be more clear for maintainers 👍  Going to add these changes